test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AccountBalancesClientSaveLoadTest.java

```diff
@@ -0,0 +1,334 @@
+import static com.hedera.services.bdd.spec.transactions.token.TokenMovement.moving;
+
+import static com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.newKeyNamed;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.exportAccountBalances;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.runWithProvider;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.sleepFor;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.freeze;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.sourcing;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+import static com.hedera.services.bdd.suites.perf.PerfUtilOps.tokenOpsEnablement;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.ACCOUNT_DELETED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.ACCOUNT_ID_DOES_NOT_EXIST;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.ACCOUNT_REPEATED_IN_ACCOUNT_AMOUNTS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.BUSY;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.DUPLICATE_TRANSACTION;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INSUFFICIENT_TOKEN_BALANCE;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_SIGNATURE;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.PLATFORM_TRANSACTION_NOT_CREATED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKENS_PER_ACCOUNT_LIMIT_EXCEEDED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKEN_NOT_ASSOCIATED_TO_ACCOUNT;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TRANSACTION_EXPIRED;
+
+
+public class AccountBalancesClientSaveLoadTest extends LoadTest  {
+	private static final Logger log = LogManager.getLogger(AccountBalancesClientSaveLoadTest.class);
+	final static int MAX_PENDING_OPS_FOR_SETUP = 10_000;
+	final static int TOTAL_ACCOUNT = 200_000;
+	final static int ESTIMATED_TOKEN_CREATION_RATE = 50;
+	final static int ESTIMATED_CRYPTO_CREATION_RATE = 500;
+	final static long MIN_ACCOUNT_BALANCE = 1_000_000_000L;
+	final static int MIN_TOKEN_SUPPLY = 1000;
+	final static int MAX_TOKEN_SUPPLY = 1_000_000;
+	final static int MAX_TOKEN_TRANSFER = 100;
+	final static int SECOND = 1000;
+	private static final Random r = new Random();
+
+	private final static int TOTAL_TEST_TOKENS = 500;
+	private final static String ACCT_NAME_PREFIX = "acct-";
+	private final static String TOKEN_NAME_PREFIX = "token-";
+
+	private static int normal_acct_created = 0;
```

Comment:
This constant name can be in upper case letters

Reply:
Forgot to mention, this is not a constant. It means to save how many accounts created each run.